rts/RtsUtils.c

```diff
@@ -372,10 +372,6 @@ void printRtsInfo(const RtsConfig rts_config) {
     mkRtsInfoPair("Host architecture",       HostArch);
     mkRtsInfoPair("Host OS",                 HostOS);
     mkRtsInfoPair("Host vendor",             HostVendor);
-    mkRtsInfoPair("Target platform",         TargetPlatform);
-    mkRtsInfoPair("Target architecture",     TargetArch);
-    mkRtsInfoPair("Target OS",               TargetOS);
-    mkRtsInfoPair("Target vendor",           TargetVendor);
```

Comment:
Don't we want to retain Build/Host/Target?

Reply:
Nope, it doesn't make sense for the RTS to know about a target, it's not a compiler.
- target: doesn't make any sense
- host: makes sense
- build: taint the program for no good reason as in theory the build platform shouldn't have an impact on the build product. So we could remove it too.

Reply:
For the RTS I agree, that one only has one arch, it's built for and runs on.

Although I'll say that I'm not sure if the rts has some weird built/host/target conditions (which it probably shouldn't have either).

Reply:
> Although I'll say that I'm not sure if the rts has some weird built/host/target conditions (which it probably shouldn't have either).

I tried to remove those long ago. I hope no one put any back! :)